nextstrain/cli/runner/conda.py

```diff
@@ -422,30 +422,27 @@ def runnable(*argv) -> bool:
 
     support = test_support()
 
+    yield from support
+
     if not runner_tests_ok(support):
-        return support
+        pass
```

Comment:
In the context of the switch to generators, this re-use of `support` surprised me and looked like a bug. It's not because `test_support()` is still returning a list instead of switching to a generator. This is a bug waiting to happen though, e.g. if someone goes, "Oh, `test_support()`'s not a generator yet, I'll fix that." I'd suggest the following small change to more can guard against that and more clearly indicate expectations.

```suggestion
    support = list(test_support())

    yield from support

    if not runner_tests_ok(support):
        return
```

(includes using `return` instead of `pass` to return early, matching existing code)

Reply:
Good catch, done:

https://github.com/nextstrain/cli/blob/f6ddf6e139fd56aa63568955901c11786717e0fd/nextstrain/cli/runner/conda.py#L423

I've also updated `test_support()` and everything else that returns `RunnerTestResults` to return a generator.

https://github.com/nextstrain/cli/blob/f6ddf6e139fd56aa63568955901c11786717e0fd/nextstrain/cli/runner/conda.py#L467-L473